mscore/scoreview.cpp

```diff
@@ -3590,9 +3590,9 @@ void ScoreView::cmdAddSlur(const Slur* slurTemplate)
                         if (!e->isChord())
                               continue;
                         ChordRest* cr = toChordRest(e);
-                        if (!cr1 || cr1->tick() > cr->tick())
+                        if (!cr1 || cr->isBefore(cr1))
                               cr1 = cr;
-                        if (!cr2 || cr2->tick() < cr->tick())
+                        if (!cr2 || cr2->isBefore(cr))
```

Comment:
these change still does apply to master, with some minor changes:

                Ms::ChordRest* cr = Ms::toChordRest(e);
                if (!firstChordRest || cr->isBefore(firstChordRest)) {
                    firstChordRest = cr;
                }
                if (!secondChordRest || secondChordRest->isBefore(cr)) {
                    secondChordRest = cr;
                }

and file location